packages/discovery-react-components/src/components/StructuredQuery/utils/stringifyStructuredQuerySelection.ts

```diff
@@ -0,0 +1,32 @@
+import { StructuredQuerySelection } from './structuredQueryInterfaces';
+
+export const stringifyStructuredQuerySelection = (
+  structuredQuerySelection: StructuredQuerySelection
+) => {
+  let stringifiedStructuredQuerySelection: string = '';
+  stringifiedStructuredQuerySelection += Object.keys(structuredQuerySelection.groups)
```

Comment:
i thought that `group_order` was going to be used here to grab the groups instead of doing `Object.keys` on the groups object?

Reply:
Ah, yep, updated...thanks for keeping me honest!